frontend/src/metabase/query_builder/components/filters/modals/BulkFilterModal/BulkFilterModal.styled.tsx

```diff
@@ -8,17 +10,18 @@ import IconButtonWrapper from "metabase/components/IconButtonWrapper";
 export const ModalRoot = styled.div`
   display: flex;
   flex-direction: column;
-  max-height: 90vh;
+  height: 90vh;
```

Comment:
modal needs a fixed height now that we potentially have a rapidly varying number of fields on the screen